apps/dashboard/src/components/workflow-editor/steps/component-utils.tsx

```diff
@@ -49,10 +50,14 @@ export const getComponentByType = ({ component }: { component?: UiComponentEnum
     case UiComponentEnum.DELAY_TYPE:
       return <DelayAmount />;
 
-    case UiComponentEnum.BLOCK_EDITOR: {
-      return <EmailBodyEditor />;
+    case UiComponentEnum.EMAIL_EDITOR_SELECT: {
+      return <EmailEditorSelect />;
     }
 
+    case UiComponentEnum.EMAIL_BODY:
+    case UiComponentEnum.BLOCK_EDITOR:
+      return <EmailBody />;
```

Comment:
`BLOCK_EDITOR` deprecated, introduced `EMAIL_BODY` which is more generic; we need to create a migration not only for this field but also for the above select, I'll add it to the TODO list